src/main/java/teammates/ui/pagedata/AdminActivityLogPageData.java

```diff
@@ -74,13 +74,19 @@ private void setDefaultLogSearchPeriod() {
         toDateValue = TimeHelper.now(0.0).getTimeInMillis();
     }
 
-    public void init(boolean ifShowAll, boolean ifShowTestData, List<ActivityLogEntry> logs) {
-        this.ifShowAll = ifShowAll;
-        this.ifShowTestData = ifShowTestData;
+    public void init(List<ActivityLogEntry> logs) {
         this.logs = logs;
 
     }
 
+    public void setIfShowAll(boolean val) {
+        ifShowAll = val;
```

Comment:
Is the `setIfShowAll` more like a setter method? (although I admit that `ifShowAll` is a bad name)